packages/vaadin-lumo-styles/utilities/shadow.js

```diff
@@ -5,8 +5,8 @@
  */
 import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
-export const shadows = css`
-  /* === Box shadows === */
+export const shadow = css`
```

Comment:
Renaming this file and exported literal is a breaking change as it might break user's apps importing it.
Consider reverting this change, or adding `shadows.js` file back with the following content:

```js
export { shadow as shadows } from './shadow.js';
```

Reply:
Understood. I'll revert it.